src/pages/UnlockedWallet/AddressesPage/AddressGridRow.tsx

```diff
@@ -59,17 +60,32 @@ const AddressGridRow = ({ addressHash, className }: AddressGridRowProps) => {
   const [isAddressDetailsModalOpen, setIsAddressDetailsModalOpen] = useState(false)
 
   const assetsWithBalance = assets.filter((asset) => asset.balance > 0)
-  const [knownAssets, unknownAssets] = partition(assetsWithBalance, (asset) => !!asset.symbol)
-
-  const [assetSymbolTexts, setAssetSymbolTexts] = useState<string[]>([])
-  const [nbOfHiddenSymbols, setNbOfHiddenSymbols] = useState(0)
 
   const assetsCellRef = useRef<HTMLDivElement>(null)
 
   if (!address) return null
 
   const fiatBalance = calculateAmountWorth(BigInt(address.balance), price ?? 0)
 
+  const renderAssetLogos = () => {
+    const [displayedAssets, hiddenAssets] = chunk(assetsWithBalance, maxDisplayedAssets)
```

Comment:
This is problematic because `assetsWithBalance` might contain in various places unknown tokens where the symbol is `undefined`. I'll update the code to fix it!